wicket-devutils/src/main/java/org/apache/wicket/devutils/debugbar/wicket-debugbar.js

```diff
@@ -26,5 +26,45 @@ function wicketDebugBarToggleVisibility(elemID) {
 	var elem = document.getElementById(elemID);
 	var vis  = elem.style.display != 'none';
 	elem.style.display = (vis ? 'none' : '');
+    // alter the state cookie so we can initialize it properly on domReady
+	wicketDebugBarSetExpandedCookie(vis ? 'collapsed' : 'expanded')
 }
 
+function wicketDebugBarSetExpandedCookie(value) {
+	document.cookie =  "wicketDebugBarState=" + window.escape(value);
+}
+
+function wicketDebugBarGetExpandedCookie() {
+	var name = 'wicketDebugBarState';
+	if (document.cookie.length > 0) {
+		var start = document.cookie.indexOf (name + "=");
+		if (start !== -1) {
+			start = start + name.length + 1;
+			var end = document.cookie.indexOf(";", start);
+			if (end === -1) {
+				end = document.cookie.length;
+			}
+			return window.unescape(document.cookie.substring(start,end));
+		} else {
+			return null;
+		}
+	} else {
+		return null;
+	}
+}
+
+function wicketDebugBarCheckState() {
+	var state = wicketDebugBarGetExpandedCookie();
+    // state cookie has not been set. determine state and set it
+	if (state === null) {
+		var isVisible = $('#wicketDebugBarContents').is(':visible');
+		wicketDebugBarSetExpandedCookie(isVisible ? 'expanded' : 'collapsed');
+    // set state of debug bar according to cookie
+	} else {
+		if (state === 'expanded') {
+			$('#wicketDebugBarContents').css('display', 'inherit');
```

Comment:
`display:inherit` leads to a display issue for me (firefox 38).
If I remove the inline `display:inherit`, it works. I admit this is weird...
It does not repro on Chrome.

![debugbar](https://cloud.githubusercontent.com/assets/1717144/7832602/8998dd50-0460-11e5-85e7-70e1dfa38b1a.png)